.github/workflows/build-release-import-script.yml

```diff
@@ -36,30 +41,31 @@ jobs:
       - run: make validate-fmt
 
   goreleaser:
+    needs: [build, lint]
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
```

Comment:
Any reason why we're switching the version?

Reply:
nope, changed